packages/url_launcher/url_launcher/README.md

```diff
@@ -133,22 +145,37 @@ due to [a bug](https://github.com/dart-lang/sdk/issues/43838) in the way `Uri`
 encodes query parameters. Using `queryParameters` will result in spaces being
 converted to `+` in many cases.
 
+<?code-excerpt "encoding.dart (encode-query-parameters)"?>
 ```dart
 String? encodeQueryParameters(Map<String, String> params) {
   return params.entries
-      .map((e) => '${Uri.encodeComponent(e.key)}=${Uri.encodeComponent(e.value)}')
+      .map((MapEntry<String, String> e) =>
+          '${Uri.encodeComponent(e.key)}=${Uri.encodeComponent(e.value)}')
       .join('&');
 }
+// ···
+  final Uri emailLaunchUri = Uri(
+    scheme: 'mailto',
+    path: 'smith@example.com',
+    query: encodeQueryParameters(<String, String>{
+      'subject': 'Example Subject & Symbols are allowed!',
+    }),
+  );
+
+  launchUrl(emailLaunchUri);
+```
 
-final Uri emailLaunchUri = Uri(
-  scheme: 'mailto',
-  path: 'smith@example.com',
-  query: encodeQueryParameters(<String, String>{
-    'subject': 'Example Subject & Symbols are allowed!'
-  }),
-);
+Encoding for `sms` is slightly different:
 
-launchUrl(emailLaunchUri);
+<?code-excerpt "encoding.dart (sms)"?>
+```dart
+final Uri smsLaunchUri = Uri(
+  scheme: 'sms',
+  path: '0118 999 881 999 119 7253',
```

Comment:
🤣 

I was about to ask why this wasn't using some obviously fake number (like 123456789) when I realized what it was 🙂

Reply:
I tried to find a cool "555" number, but ended up going for the _easy to remember_ 0118 999 881 999 119 725... 3

_(Turns out I'm [almost 7 years late](https://www.reddit.com/r/ITcrowd/comments/3ses6o/it_crowd_easter_egg_i_found_on_my_android_phone/), yikes!)_